content/03.methods.md

```diff
@@ -122,19 +122,19 @@ The cancer type-specific baseline models were the same, without the cancer type
 
 To predict mutation presence or absence in cancer genes using multiple data types simultaneously, we concatenated individual datasets into a large feature matrix, then used the same elastic net logistic regression method described previously.
 For this task, we considered only the gene expression, 27K methylation, and 450K methylation datasets.
-We used only these data types to limit the number of multi-omics combinations: the expression and methylation datasets resulted in the best overall performance across the single-omics experiments so we limited combinations to those datasets.
-In the main text, we report results using the top 5,000 principal components for each dataset, to ensure that the information content of the included features is comparable between data types.
+We used only these data types to limit the number of multi-omics combinations; the expression and methylation datasets resulted in the best overall performance across the single-omics experiments, so we limited combinations to those datasets.
+In the main text, we report results using the top 5,000 principal components for each dataset to ensure that the information content of the included features is comparable between data types.<!--The top 5000 wouldn't necessarily capture the same amount of variance across different experiments, though, would they? It's more just that that is a lot of PCs so you're capturing quite a bit of variance?-->
```

Comment:
```suggestion
In the main text, we report results using the top 5,000 principal components for each dataset to ensure that most variance is captured (approximately 95-98% of variance for each data type).
```
I looked at this in greenelab/mpmp#19, and it does turn out that the amount of variance captured by 5000 PCs is about the same for expression and methylation (although not exactly the same). What you said is also true, though - we picked 5000 because it's likely that this many PCs will capture most of the useful signal.

Reply:
That definitely makes sense! I think it was the "to ensure" that was tripping me up (to me it implies that there's something I'm supposed to know about this). So I would think of what you are saying as something like "we report results using the top 5,000 principal components for each dataset, which ensures that most variance is captured (approximately 95-98% of variance for each data type)."